stor/base.py

```diff
@@ -388,12 +392,12 @@ def listdir(self):
 
         The elements of the list are Path objects.
         """
-        return [self / child
+        return [old_div(self, child)
                 for child in map(self._always_unicode, os.listdir(self))]
 
     def glob(self, pattern):
         cls = self.path_class
-        return [cls(s) for s in glob.glob(self / pattern)]
```

Comment:
Presumably we should disable that flag and do division changes manually given the abundance of string division.